### DISCUSSION TITLE
Pin XX is used in multiple places.

### DISCUSSION THREAD

**Discussion by ivercellino** [dbuezas/esphome-cc1101#20] [Q&A] [answered]
I'm using this custom sensor with a 868 Mhz module and it's working fine.
But when try to enable both trnasmitter and receiver ESPHome raise an error saying that the same pin is used in multiple places and do not compile.

Any tips to fix the issue?

Thanks in advance.

**Comment by dbuezas** [comment, marked as answer]
https://github.com/dbuezas/esphome-cc1101/issues/17#issuecomment-1866707558

If you could test it and open a pr, I'd be thankful. I'm on my phone for a while and you are the third facing this :)
